packages/bootstrap/extra/dcos_internal_utils/exhibitor/exhibitor.py

```diff
@@ -0,0 +1,43 @@
+import codecs
+import json
+import logging
+import os
+import urllib.request
```

Comment:
I'd rather we just use requests (All the rest of our code was ported to requests only)